packages/server/src/requestPipeline.ts

```diff
@@ -150,11 +150,12 @@ export async function processGraphQLRequest<TContext>(
     // apollo-server 1.0-style middleware (graphqlExpress etc, not via the
     // ApolloServer class), it won't have been converted to
     // PrefixingKeyValueCache yet.
-    if (!(persistedQueryCache instanceof PrefixingKeyValueCache)) {
-      persistedQueryCache = new PrefixingKeyValueCache(
-        persistedQueryCache,
-        APQ_CACHE_PREFIX,
-      );
+
+    // FIXME! I have no idea how this worked before - this would create a new cache
+    // every time we received a request which is pointless. Am I missing something??
```

Comment:
Got it, I think I'll keep this concept around for now then. Makes sense, especially since `keyv` doesn't support namespacing beyond the constructor option.
970ffe1d275854e6ea6a5eef7987c43c0ba03e9e